apps/blazor-test-app/Pages/_Host.cshtml

```diff
@@ -0,0 +1,36 @@
+@page "/"
+@namespace Blazor_Test_App.Pages
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+@{
+    Layout = null;
+}
+
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Blazor-Test-App</title>
+    <base href="~/" />
+    <link href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css" rel="stylesheet"/>
+    <link href="css/site.css" rel="stylesheet" />
+    <link href="Blazor-Test-App.styles.css" rel="stylesheet" />
+</head>
+<body>
+    <component type="typeof(App)" render-mode="ServerPrerendered" />
+
+    <div id="blazor-error-ui">
+        <environment include="Staging,Production">
+            An error has occurred. This application may no longer respond until reloaded.
+        </environment>
+        <environment include="Development">
+            An unhandled exception has occurred. See browser dev tools for details.
+        </environment>
+        <a href="" class="reload">Reload</a>
+        <a class="dismiss">🗙</a>
+    </div>
+
+    <script src="_framework/blazor.server.js"></script>
+    <script type="module" src="https://unpkg.com/@@fluentui/web-components"></script>
```

Comment:
What is `unpkg.com` ? Is it stable registry? Why not npm registry?

Reply:
*Note: this file was auto-generated by the TeamsFX toolkit* 
unpkg.com is a third party, open-source CDN for npm packages. The TeamsFX team seems to prefer it as their CDN of choice for (at least) their Blazor applications. I know it is very fast and reliable, but mostly I wanted to keep the CDN consistent with other Blazor apps customers would have, so I didn't want to change it to npm.

Reply:
I would suggest it to have `npm` packages, we cannot rely on 3rd party CDNs.

Reply:
So Blazor also just outright does not support using npm. I found a hacky way of possibly getting it to work, but it would likely be far less reliable than just using unpkg. The npm documentation for the fluentui/web-components package actually uses unpkg. I also saw it recommended by other microsoft employees in the github threads I was looking at to try and find a way to get it working with npm, so leaving it using unpkg will be fine.

Reply:
Given that the goal of this app is to accurately represent what a customer would do to avoid breaking them, using the Teams Toolkit defaults is the right thing to do here.